memstore/src/lib.rs

```diff
@@ -164,7 +164,7 @@ impl RaftStorage<ClientRequest, ClientResponse> for MemStore {
     }
 
     #[tracing::instrument(level = "debug", skip(self, range), fields(range=?range))]
-    async fn delete_logs_from<R: RangeBounds<u64> + Clone + Debug + Send + Sync>(
+    async fn delete_log<R: RangeBounds<u64> + Clone + Debug + Send + Sync>(
```

Comment:
how about remain plural `delete_log`->`delete_logs`

Reply:
Ok

Reply:
Will be fixed in next PR:DDD